src/main/resources/application.yaml

```diff
@@ -1,22 +1,39 @@
-server:
-  port: 4458
-
-management:
-  endpoint:
-    health:
-      show-details: "always"
-  endpoints:
-    web:
-      base-path: /
-      exposure:
-        include: health, info, prometheus
-
-#If you use a database then uncomment below lines and update db properties accordingly
 spring:
   config:
-    import: "optional:configtree:/mnt/secrets/"
+    import: "optional:configtree:/mnt/secrets/ccd/"
   application:
     name: Case Disposer
+  jpa:
+    database: postgresql
+    show-sql: ${SHOW_SQL:true}
+    hibernate:
+      ddl-auto: none
+    database-platform: org.hibernate.dialect.PostgreSQL95Dialect
+    properties:
+      hibernate:
+        temp.use_jdbc_metadata_defaults: false
+        order_inserts: true
+        order_updates: true
+        jdbc:
+          batch_size: 5
+          batch_versioned_data: true
+  datasource:
+    url: jdbc:postgresql://${DATA_STORE_DB_HOST:localhost}:${DATA_STORE_DB_PORT:5452}/${DATA_STORE_DB_NAME:ccd_data}${DATA_STORE_DB_OPTIONS:?stringtype=unspecified}
+    username: ${DATA_STORE_DB_USERNAME:ccd}
+    password: ${DATA_STORE_DB_PASSWORD:ccd}
+    type: com.zaxxer.hikari.HikariDataSource
+    hikari:
+      connection-timeout: ${DATA_STORE_DB_CONNECTION_TIMEOUT:40000}
+      idle-timeout: ${DATA_STORE_DB_IDLE_TIMEOUT:300000}
+      minimum-idle: ${DATA_STORE_DB_MIN_IDLE:8}
```

Comment:
we probably need only one connection. Could you check if there is an option to use SingleConnection datasource etc.